static/js/popover_menus.js

```diff
@@ -87,4 +107,196 @@ export function initialize() {
             compose_mobile_button_popover_displayed = false;
         },
     });
+
+    // Message actions popover.
+    delegate("body", {
+        target: ".actions_hover i.zulip-icon-ellipsis-v-solid",
```

Comment:
Why not just `actions_hover`, as was the previous selector?  I'm worried that this might unexpectedly shrink the click area.

Reply:
I've confirmed this is in fact a bug -- you can get the hover behavior to show where clicking does nothing with this revised selector.

Reply:
Tippy cannot handle the same element having a  popover and tooltip. So, I had to use the icon instead for the popover. This is an issue we will see while porting all the popovers. I tried to increase the clickable area, as you have noticed below. Let me see if I can improve it.

Reply:
Hmm, annoying.  Can you start a #frontend thread about this?  I'd like to figure out a pattern for how to manage that kind of situation, since it's going to happen a lot as we port our popovers.